src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/TransitLayer.java

```diff
@@ -48,7 +48,11 @@ public class TransitLayer {
 
   private final TransferIndexGenerator transferIndexGenerator;
 
-  private final int[] stopBoardAlightCosts;
+  /**
+   * @see #getStopBoardAlightTransferCosts()
+   */
```

Comment:
By convention we add JavaDoc on the public part of a class API, so no need to link to it here.
```suggestion
```